examples/oauth/06-settlements.py

```diff
@@ -0,0 +1,82 @@
+# coding=utf-8
+
+from __future__ import print_function
+
+from mollie.api.error import Error
+
+
+def main(client):
+    try:
+        body = ''
+
+        # https://docs.mollie.com/reference/v2/settlements-api/list-settlements
+
+        body += '<h1>List settlements</h1>'
+        response = client.settlements.list()
+
+        print(response)
+        body += str(response)
+
+        settlement_id = 'stl_123'  # Your open settlement id here
```

Comment:
It would be cleaner to retrieve the "open" settlement first, then then use it's actual ID to demonstrate getting by ID.

Reply:
@kingel I need your help testing this. I doubt that we have any settlements at all.

Reply:
Done, I get the first settlement_id here form list. 

Note: The 'open' endpoint returns an object with `id:'open'`. Not very useful, if you ask me.